bookie/views/auth.py

```diff
@@ -82,6 +83,12 @@ def login(request):
             message = "Failed login"
             AuthLog.login(login, False, password=password)
 
+    if isinstance(request.user, User):
```

Comment:
I'd like this to be a bit smarter. What about not just checking there is a User, but that it's a current valid user in the system that is activated as a security measure?